docs/logging-stdlib-usage.rst

```diff
@@ -0,0 +1,71 @@
+Integration with Python logging module
+--------------------------------------
+
+
+It's possible to tie the Python :mod:`logging` module directly into Google Cloud Logging. To use it,
+create a :class:`CloudLoggingHandler <google.cloud.logging.CloudLoggingHandler>` instance from your
```

Comment:
@tseaver All of the changes I asked for are teeny. Are there any in particular that you disagree with or think are difficult to implement? Most of them (all of them?) are just "delete this content" or "use a `~`".